### ISSUE TITLE
Implement the CMOR & ece2cmor3 injections for the extremeX project

### ISSUE THREAD

**Issue by treerink** [EC-Earth/ece2cmor3#717]
The extremeX project requires a lot of non standard CMOR output, however the project wish is to output all output along CMOR standard where possible and in the CMOR way for the remaining part.

Therefore an `add-variables-for-extremeX.sh` script needs to be written which can inject this required additions for cmorising non-CMOR existing extremeX output into the CMOR tables and for inserting additional variables in the `ifspar.json`. Note that the list of injections / patches is rather extensive.

With this work the extremeX additions will be maintainable and archived, thus also easier available later. The direct motivation however, is the fact that these additions have been made in a very old `ece2cmor3` `v1.2` version, and recmorisation with a updated `ece2cmor3` version including the updated CMOR tables is inevitable.

**Comment by treerink** [comment]
Script is ready, needs validation now.

**Comment by treerink** [comment]
A first successful `extremeX` cmorisation case has been achieved.

**Comment by treerink** [comment]
The 9f4ea1f fix for #797 broke the `add-variables-for-extremeX.sh` script, leading to the cmorisation errors:
```
s001-ifs-001-20240426102423.log:ERROR:ece2cmor3.taskloader: Variable zg850 in table day  is not supported by ifs in ece2cmor3; if you do expect an ec-earth output variable here, please create an issue or pull request on our github page
s001-ifs-001-20240426102423.log:ERROR:ece2cmor3.taskloader: Variable zg200 in table day  is not supported by ifs in ece2cmor3; if you do expect an ec-earth output variable here, please create an issue or pull request on our github page
s001-ifs-001-20240426102423.log:ERROR:ece2cmor3.taskloader: Variable zg9   in table Amon is not supported by ifs in ece2cmor3; if you do expect an ec-earth output variable here, please create an issue or pull request on our github page
```
I will fix & test.

**Comment by treerink** [comment]
Test is ok, the following:
```
day zg850
day zg200
Amon zg    # coming from zg9
```
are produced again. So with 0e5f771 the latest issue is solved.
